benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -173,6 +173,7 @@ export default class Overview extends React.Component {
                     filtered={this.state.filtered}
                     toggleLinkOverlay={this.toggleLinkOverlay}
                     changeTab={this.changeTab}
+                    hasAccessToUrlParams={!this.state.showSelectColumns}
```

Comment:
I am not sure why this is necessary? In general, more comments would be good, in particular for cases like this one.

Reply:
This was necessary due to the `SelectColumn` and `ReactTable` components both having a popstate event-listener while operating on the same page. This resulted in unwanted side effects, so this flag was for deactivating the popstate event in the `ReactTable` component while the `SelectColumn` is open. However this solution will not be necessary any more after removing the `isVisible` flag of the columns object any way. I will check where to add commentary for the following changes though.